Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM python:2.7.12
+
+COPY ./docker/prod/environment /etc/consul_template_env.sh
+
+COPY ./docker/prod/docker-helpers/install_consul_template.sh \
+     ./docker/prod/docker-helpers/install_runit.sh \
+     /usr/local/bin/
+RUN chmod 755 /usr/local/bin/install_consul_template.sh /usr/local/bin/install_runit.sh && \
+    sync && \
+    install_consul_template.sh && \
+    rm -f \
+        /usr/local/bin/install_consul_template.sh \
+        /usr/local/bin/install_runit.sh
+
+################
+# CAA Redirect #
+################
+
+RUN mkdir /code
+WORKDIR /code
+
+# Python dependencies
+RUN apt-get update && \
+    apt-get install -y \
+            build-essential \
```

Comment:
We can uninstall all these at the end if they're only needed to build the image. As long as the runtime libs stay. I store the package lists in a variable in https://github.com/metabrainz/musicbrainz-server/blob/docker/Dockerfile so I can add/remove things as a group.